README.md

```diff
@@ -101,3 +101,77 @@ Here are some resulting renders from the demo application.
 **BarramundiFish Model**
 
 ![](images/BarramundiFish.gif)
+
+Appendix
+------------
+
+In this section, you'll find alternative implementations for the various terms found in the lighting equation.
+
+### Surface Reflection Ratio (F)
+
+**Frensel Schlick**
+Simplified implementation of fresnel from "An Inexpensive BRDF Model for Physically based Rendering" by Christophe Schlick.
```

Comment:
Are there reasonable links for these papers? If so it would be nice to link to them directly to save a Googling. :)